src/App.tsx

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+const App: React.FC = () => {
+  const [weatherData, setWeatherData] = useState<any>(null);
```

Comment:
Please avoid using any types, as they undermine TypeScript's type safety. Instead, take a closer look at the data in your weatherData state and define a specific interface that accurately describes its structure. If you need assistance, feel free to use ChatGPT for help with type definitions.